### DISCUSSION TITLE
There is a way to bypass having to be host to do certain things

### DISCUSSION THREAD

**Discussion by Lchaxlover** [winstxnhdw/lc-hax#488] [General]
Yesterday I met a homebrewer on lc and he was able to do anything in my lobby. He ejected us and spawned enemies (other than masks). I dont know how he did it i tried to ask but he was a jackass so didnt tell me. But it is possible!

**Comment by winstxnhdw** [comment]
There is no use in telling me this. I'll need a source of any sort. It's the same as telling someone that the Boogieman exists and expecting them to investigate it.

**Reply by Lchaxlover** [reply]
lol

**Reply by Lchaxlover** [reply]
alr well i just wanted to let you know its possible

**Reply by winstxnhdw** [reply]
I think you have already done that in the other discussion.

**Reply by Lchaxlover** [reply]
oh and one more thing, you should add an eject lobby command

**Reply by Lchaxlover** [reply]
Like when you dont meet quota how you get fired, that thing but do it whenever

**Reply by winstxnhdw** [reply]
It's host-only.

**Reply by Lchaxlover** [reply]
Ik but you should still add it

**Reply by winstxnhdw** [reply]
You can just type `eject` in your terminal.

**Reply by Lchaxlover** [reply]
Without cheats?

**Reply by winstxnhdw** [reply]
Yeah?

**Reply by Lchaxlover** [reply]
bruh how did i not know this

**Reply by winstxnhdw** [reply]
Skill issue

**Reply by Lchaxlover** [reply]
real.

**Reply by Lchaxlover** [reply]
btw the homebrewer was talking mad shit on your hacks

**Reply by Lchaxlover** [reply]
Idk why there the best hacks online rn

**Reply by winstxnhdw** [reply]
I am happy for him 👍🏼

**Reply by Lchaxlover** [reply]
im trying to pay someone on fiverr to see if they can find out how to bypass host commands lol

**Reply by Lchaxlover** [reply]
If they can I will give you the info how

**Reply by winstxnhdw** [reply]
Your dedication will be remembered 🔥

**Reply by Lchaxlover** [reply]
Im not sure if you can tell but I really want to bypass host commands lol

**Reply by Lchaxlover** [reply]
Yesterday I did a bit in lc where i named myself lethal jesus and when I joined a lobby i used a sound board to have the heaven music go ahhhhh ahhhh while i was floating telling them to kneel before me and if they didnt i would play a tts message telling them their server was going to be destroyed, then after the message i played insanely loud Scottish music and did a bunch of shit to their server lol

**Reply by winstxnhdw** [reply]
joep would be proud of u

**Reply by Lchaxlover** [reply]
who's joep

**Reply by winstxnhdw** [reply]
joep mama

**Reply by joep26020** [reply]
i am very proud lol thats pretty good

**Comment by Lchaxlover** [comment]
Do you use java for the hacks? Im going to have to ask you some questions about the hacks so my guy knows what he is working with

**Reply by winstxnhdw** [reply]
Why not just send him the link to this GitHub repository?

**Reply by Lchaxlover** [reply]
Oh good. idea im starting to notice that i am a dumbass

**Reply by winstxnhdw** [reply]
average lethal company player

**Comment by joep26020** [comment]
and how do you for sure know he's not the host?
